### PR TITLE
rapidfuzz 2.13.7

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Changelog: https://github.com/maxbachmann/RapidFuzz/blob/v2.13.7/CHANGELOG.rst
License:
Requirements:
- https://github.com/maxbachmann/RapidFuzz/blob/v2.13.7/CMakeLists.txt
- https://github.com/maxbachmann/RapidFuzz/blob/v2.13.7/pyproject.toml
- https://github.com/maxbachmann/RapidFuzz/blob/v2.13.7/setup.py

Actions:
1. Skip py<37
2. Add --no-deps
3. Remove cross-platfrom dependencies from build
4. Remove rapidfuzz_capi from host
5. Remove jarowinkler from run
6. Add rapidfuzz.distance to test/imports
7. Set Visual Studio to vs2019

Notes:

- cython was removed from sdist, see https://github.com/maxbachmann/RapidFuzz/blob/ad632b078fcb440e4e073cf03065a78b020f01e8/CHANGELOG.rst#2012---2022-06-22 and https://github.com/maxbachmann/RapidFuzz/blob/3d1ccd389f87ca0fd10010a2d694145d09a0702a/tools/sdist.patch

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,41 +1,46 @@
...
+  sha256: 8d3e252d4127c79b4d7c2ae47271636cbaca905c8bb46d80c7930ab906cf4b5c
+  patches:
+    - cmake_verfix.patch
 
 build:
   number: 0
   skip: True  # [py<37]
-  script:
-    - {{ PYTHON }} -m pip install . -vvv
-  missing_dso_whitelist:  # [s390x]
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:17]:
```
  missing_dso_whitelist:  # [s390x]
    - '$RPATH/ld64.so.1'  # [s390x]
```
It's still required for s390x. I'll return it back

Comment (katietz) [reply recipe/meta.yaml:17]: 👍

**Review by katietz — APPROVED** [review]
lgtm, thanks!

**Review by varlackc — APPROVED** [review]
As soon as we are able to fix the issue in win-64 we should be able to approve.

**Review by ELundby45 — APPROVED** [review]

**Review by SC426 — APPROVED** [review]

**Comment by skupr-anaconda** [comment]
@cbouss Thank you!

**Pull request merged**